backend/resources/auth.py

```diff
@@ -42,21 +42,21 @@ def post(self):
                 HTTPStatus.BAD_REQUEST,
             )
 
-        username = result["username"]
+        github = result["github"]
         password = result["password"]
 
-        if not (username and password):
-            return ({"msg": "Username and password required"}, HTTPStatus.BAD_REQUEST)
+        if not (github and password):
+            return ({"msg": "github and password required"}, HTTPStatus.BAD_REQUEST)
 
-        user = User.query.filter_by(username=username).first()
+        user = User.query.filter_by(github=github).first()
 
         if user and user.check_password(password):
             access_token = create_access_token(
-                identity=username, expires_delta=timedelta(minutes=60)
+                identity=github, expires_delta=timedelta(minutes=60)
             )
 
             refresh_token = create_refresh_token(
-                identity=username, expires_delta=timedelta(weeks=1)
+                identity=github, expires_delta=timedelta(weeks=1)
```

Comment:
Does this require adjustments on the frontend? That is replacing e-mail for github nick for current version of the login form?

Reply:
I will need to check that.

Reply:
Fixed in commit: https://github.com/CodeForPoznan/codeforpoznan.pl_v3/pull/407/commits/5c5b671a3929e4799919916d80c71f577aa1d3ff